Week-11/Redux-Exercise1/theme-switcher/src/components/Room/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { useTheme } from "app/hooks";
+
+const Room = () => {
+  const theme = useTheme();
+
+  return <>{theme === "dark" ? "the room is dark" : "the room is lit"}</>;
```

Comment:
@apandyamob Typo error for lit, i think its light. You can make it as same everyone for consistency

Reply:
fixed @RajeshSivanesan , thanks !